types/test_utils.go

```diff
@@ -36,7 +36,7 @@ func GetRandomSignedHeader() (*SignedHeader, ed25519.PrivKey, error) {
 
 			BaseHeader: BaseHeader{
 				ChainID: "test",
-				Height:  rand.Uint64(), //nolint:gosec,
+				Height:  uint64(rand.Int63()), //nolint:gosec,
```

Comment:
I'm confused at what is happening here. Why don't we continue to use `rand.Uint64`?

Reply:
If we use `rand.Uint64` and get a number bigger than the positive range of int64, converting it to int64 will make it go negative, so better to stick to `rand.Int63()` which `returns a non-negative pseudo-random 63-bit integer as an int64`